### PR TITLE
Fix docs build languange tests

### PR DESCRIPTION
**Pull request by clay-lake**
This PR includes several fixes for the language tests in our docs/Makefile:

- Fixed woke utility skipping .*ignore files. This appears to be a bug in the utility. I'll raise this issue with the maintainer.
- Disabled default woke configuration and exclusively test ours in canonical/Inclusive-naming.
- Enabled error exit status. The Tests action will now show if there are language issues.

- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

**Comment by clay-lake** [comment]
Here is an example of a failed build with these changes: https://github.com/canonical/rockcraft/pull/643

File: docs/Makefile [hunk 1]
```diff
@@ -39,7 +43,14 @@ linkcheck:
 
 woke:
 	type woke >/dev/null 2>&1 || { snap install woke; exit 1; }
-	woke *.rst **/*.rst -c https://github.com/canonical-web-and-design/Inclusive-naming/raw/main/config.yml
+
+	# Woke appears to have a issue where it skips .*ignore files unless it is executed from the repo root.
```
Comment (cjdcordeiro) [inline docs/Makefile:47]:
```suggestion
	# Woke appears to have an issue where it skips .*ignore files unless it is executed from the repo root.
```

**Review by cjdcordeiro — APPROVED** [review]
Interesting. Thanks ! just a small typo but it lgtm

File: docs/Makefile [hunk 2]
```diff
@@ -39,7 +43,14 @@ linkcheck:
...
+	# This is mitigated by changing to the repo root and running woke on the relative docs path.
+	# Tested on 19/07/24 with woke 0.19.0-4-g5d52c15
+	cd ..; woke $(RELCWD)/**/*.rst $(RELCWD)/*.rst \
```
Comment (cjdcordeiro) [inline docs/Makefile:50]:
```suggestion
	cd ${REPOROOT}; woke $(RELCWD)/**/*.rst $(RELCWD)/*.rst \
```
that's safer perhaps?

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/Makefile (hunk 1 above)
Comment (clay-lake) [reply docs/Makefile]: Good eye! Thanks

File: docs/Makefile (hunk 2 above)
Comment (clay-lake) [reply docs/Makefile]: I like it. That is more explicit as well. I pushed the change.

**Review by cjdcordeiro — APPROVED** [review]
Thanks. The spread error you're getting is most likely because of this: https://github.com/canonical/rockcraft/pull/645

**Comment by clay-lake** [comment]
> Thanks. The spread error you're getting is most likely because of this: #645

Could this be it?... it looks like a slice definition changed in chisel-releases which broke a test we run on our tutorials.
In particular, the tutorial copies in its own openssl.yaml which is missing a data slice now required by ca-certificates_data-no-certs.

2024-07-22 14:40:26 Error executing google:ubuntu-22.04-64:docs/how-to/code/create-slice (jul221423-566334) : 
```
+ apt show openssl

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Package: openssl
Version: 3.0.2-0ubuntu1.16
Priority: important
Section: utils
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Debian OpenSSL Team <pkg-openssl-devel@alioth-lists.debian.net>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 2102 kB
Depends: libc6 (>= 2.34), libssl3 (>= 3.0.2-0ubuntu1.2)
Suggests: ca-certificates
Homepage: https://www.openssl.org/
Task: minimal, server-minimal
Download-Size: 1186 kB
APT-Manual-Installed: no
APT-Sources: http://us-east1.gce.archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages
Description: Secure Sockets Layer toolkit - cryptographic utility
 This package is part of the OpenSSL project's implementation of the SSL
 and TLS cryptographic protocols for secure communication over the
 Internet.
 .
 It contains the general-purpose command line binary /usr/bin/openssl,
 useful for cryptographic operations such as:
  * creating RSA, DH, and DSA key parameters;
  * creating X.509 certificates, CSRs, and CRLs;
  * calculating message digests;
  * encrypting and decrypting with ciphers;
  * testing SSL/TLS clients and servers;
  * handling S/MIME signed or encrypted mail.

+ local_release=/root/chisel-releases_create-slice
+ local_root=/root/out_create-slice
+ rm -fr /root/chisel-releases_create-slice /root/out_create-slice
+ git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases /root/chisel-releases_create-slice
Cloning into '/root/chisel-releases_create-slice'...
+ mkdir /root/out_create-slice
+ cp openssl.yaml /root/chisel-releases_create-slice/slices
+ snap install chisel --channel=latest/candidate
snap "chisel" is already installed, see 'snap help refresh'
+ chisel cut --release /root/chisel-releases_create-slice --root /root/out_create-slice openssl_bins
2024/07/22 14:40:26 Processing /root/chisel-releases_create-slice release...
error: ca-certificates_data-no-certs requires openssl_data, but slice is missing
```

**Review by tigarmo — APPROVED** [review]
thanks!

**Pull request merged**